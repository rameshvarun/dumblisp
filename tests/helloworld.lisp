; A basic Hello World example
(puts "Hello World!")
(puts "This is dumblisp.")
