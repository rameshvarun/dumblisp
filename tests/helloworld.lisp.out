Hello World!
This is dumblisp.
